src/resources/views/edit.blade.php

```diff
@@ -24,7 +24,14 @@
 
 		@include('crud::inc.grouped_errors')
 
-		  {!! Form::open(array('url' => $crud->route.'/'.$entry->getKey(), 'method' => 'put', 'files'=>$crud->hasUploadFields('update', $entry->getKey()))) !!}
+		  <form method="post"
```

Comment:
method shouldn't be PUT here like it was before ?

Reply:
@iMokhles no, method should be POST here, please see https://laravel.com/docs/5.6/routing#form-method-spoofing

Reply:
Yeah, I got stuck on this one too. the following like makes sure Laravel interprets this POST request as PUT, it's the only way I got it to work.